.github/workflows/deploy-API.yml

```diff
@@ -53,4 +53,4 @@ jobs:
           context: .
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
-          push: ${{ startsWith(github.ref, 'refs/tags/v') || endsWith(github.ref, '/stage') || endsWith(github.ref, '/main')  }}
+          push: ${{ startsWith(github.ref, 'refs/tags/v') || endsWith(github.ref, '/stage') || endsWith(github.ref, '/main')  }}
```

Comment:
why do you have changes in this file?

Reply:
because one of DevOps engineers advice me, to checks aren't performed twice